### PR TITLE
fix(fms): fix speed constraint handling during missed approach

### PR DESCRIPTION
**Pull request by BlueberryKing**
Fixes #[issue_no]

## Summary of Changes

Fixes a few issues with speed constraint handling during the missed approach.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

This PR targets the fms-v2 branch and does not require normal QA.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js [hunk 1]
```diff
@@ -1183,8 +1180,11 @@ class FMCMainDisplay extends BaseAirliners {
                         vPfd = speed;
                         this.managedSpeedTarget = speed;
                     } else {
-                        vPfd = this.computedVgd;
-                        this.managedSpeedTarget = this.computedVgd;
+                        const speedConstraint = this.getSpeedConstraint();
+                        const speed = Math.min(this.computedVgd, speedConstraint);
```
Comment (tracernz) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js:1184]: Speed constraints should only be applied in NAV mode, right?

Comment (BlueberryKing) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js:1184]: Yeah, `this.getSpeedConstraint()` should check for that.

**Review by Benjozork — APPROVED** [review]

**Comment by beheh** [comment]
@BlueberryKing Missing QA test instructions.

**Comment by BlueberryKing** [comment]
> @BlueberryKing Missing QA test instructions.

Yeah, the PR targets the fms-v2 branch, so it should not be part of the normal QA workflow. I will note this in the instructions.

**Comment by beheh** [comment]
Does this even need QA then? Or should one of the fms-v2 comitters just merge this whenever? I suspect fms-v2 itself will get a big QA push before it's merged.

**Comment by 2hwk** [comment]
lint appears to be failing for eslint?

**Comment by BlueberryKing** [comment]
> lint appears to be failing for eslint?

It's failing lint because fms-v2 is failing lint.

**Pull request merged**